lib/node_modules/@stdlib/number/float64/examples/index.js

```diff
@@ -19,6 +19,16 @@
 'use strict';
 
 var objectKeys = require( '@stdlib/utils/keys' );
+var number2json = require( './../to-json/lib' );
```

Comment:
You can revert these changes. Package examples should be place in the `examples/index.js` file local to the package, not in the namespace examples files.

Reply:
@Pranavchiku Would you mind removing these changes? This is the example file for the parent namespace, not for the package.